frontend/src/components/assessments/assessment-creation/DisplayQuestion.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {
+  Box,
+  Checkbox,
+  CheckboxGroup,
+  Input,
+  Radio,
+  RadioGroup,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+
+import { QuestionComponentResponse } from "../../../APIClients/types/TestClientTypes";
+import { QuestionElementType } from "../../../types/QuestionTypes";
+
+interface DisplayQuestionProps {
+  questionComponents: QuestionComponentResponse[];
+}
+
+const DisplayQuestion = ({
+  questionComponents,
+}: DisplayQuestionProps): React.ReactElement => {
+  const stackProps = {
+    alignItems: "left",
+    gap: "1",
+    paddingRight: "4",
+    paddingTop: "2",
+    ml: "5",
+  };
```

Comment:
could you reach out to design and ask them about spacing? the spacing between the text components and answer components seem a bit large to me & i noticed that there are no specifications in the Figma on what this should look like